### ISSUE TITLE
multiple backlights

### ISSUE THREAD

**Issue by Leandros** [limejelly/Backlight-for-XCode#35]
still not fixed.

Steps to reproduce:

- Open assistant editor
- Focus on pane
- Focus a completely different window
- Click in the other pane

**Comment by Leandros** [comment]
Related to #27

**Comment by tyeen** [comment]
Yes, the fix of #27 is not enough. Thanks for the report. Observing the notifications is not a good idea. Hope this #39 can fix it.

**Comment by limejelly** [comment]
#39 merged
please confirm that is fixed
